src/main/scala/io/citrine/lolo/linear/MultiTaskLinearRegression.scala

```diff
@@ -0,0 +1,49 @@
+package io.citrine.lolo.linear
+
+import io.citrine.lolo.api.{MultiTaskLearner, MultiTaskModel, MultiTaskTrainingResult, ParallelModels, TrainingRow}
+import io.citrine.random.Random
+
+case class MultiTaskLinearRegressionLearner(regParam: Option[Double] = None, fitIntercept: Boolean = true)
+    extends MultiTaskLearner {
+
+  override def train(
+      trainingData: Seq[TrainingRow[Vector[Any]]],
+      rng: Random
+  ): MultiTaskLinearRegressionTrainingResult = {
+    val rep = trainingData.head
+    val repLabels = rep.label
+    val numLabels = repLabels.length
+    assert(
+      repLabels.forall(_.isInstanceOf[Double]),
```

Comment:
We still have a bit of a failure in the `MultiTaskLearner` interface such that it expects a learner to act on a label `Vector[Any]`, but this one is only defined for multi-task regression.

I don't want to go through that refactor right now so just going to define as this.